text/0022-remotes.md

```diff
@@ -0,0 +1,98 @@
+- Feature Name: Remotes
+- Start Date: <!-- (fill me in with today's date, YYYY-MM-DD) -->
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Remotes act as a way for any user of Qri to setup their own server that keeps datasets alive, providing availability and ownership over data within a set of nodes that they control.
+
+# Motivation
+[motivation]: #motivation
+
+Multiple users have requested a way to keep datasets alive and available within their own network. Currently, we have a public Registry that serves a similar purpose, but it acts too centralized and is also not designed to be duplicated and deployed by existing users. Although IPFS may keep data blocks alive due to its distributed nature, there's no guarantee to keep data around forever unless it is pinned, and the pinning node remains online. Remotes solve this problem by giving control to users, letting them own their data. Having it work as a do-it-yourself service goes a long way torwards turning Qri into a descentralized (as well as distributed) service.
+
+Relatedly, the concept of the Registry, as it is now, should be deprecated and eventually removed. What is acting as the Registry now should be moved over to a normal Qri backend node that is acting as a remote.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Some definitions used in this document:
+
+* Registry - the Qri-run service for pinning, searching, etc
+* Remote - the future functionality included in the main Qri binary
+* Client - a Qri command-line program that get commands from stdin
+* Backend - a Qri program that is not the Electron frontend
+
+Currently, the Registry exists to solve multiple problems centered around a distributed system's lack of centralization:
+
+* User identity and registration
+* Search
+* Availability
+
+The Registry acts in a "semi-centralized" way: it provides these types of features, but is not strictly required to use Qri by itself. However, there is currently no way for an average user to run something like the Registry on their own.
+
+Currently functionality that exists only in the Registry now includes:
```

Comment:
I think everything from here to the end of the "Guide level explanation" section should move into the "Reference Level" section.